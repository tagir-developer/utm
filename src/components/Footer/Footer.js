import React from 'react'

const Footer = props => {

	const footerImage = {
		backgroundImage: 'url(./images/footer.jpg)'
	}

	return (
		<div className="bg-primary text-dark py-4 px-4 text-center"  style={footerImage}>
			<h4 className="display-5 text-light py-4 text-right">Генератор UTM-меток</h4>
			<p className="text-light text-right pb-1">Все права защищены © 2020</p>
		</div>
	)
}

export default Footer