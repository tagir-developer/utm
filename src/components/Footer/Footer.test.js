import React from 'react'
import Footer from './Footer'

describe('<Toggle />', () => {

	it('Should Toggle component render', () => {

		const wrapper = shallow(<Footer />)
		expect(toJson(wrapper)).toMatchSnapshot()

	})


})