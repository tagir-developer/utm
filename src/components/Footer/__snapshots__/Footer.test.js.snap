// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Toggle /> Should Toggle component render 1`] = `
<div
  className="bg-primary text-dark py-4 px-4 text-center"
  style={
    Object {
      "backgroundImage": "url(./images/footer.jpg)",
    }
  }
>
  <h4
    className="display-5 text-light py-4 text-right"
  >
    Генератор UTM-меток
  </h4>
  <p
    className="text-light text-right pb-1"
  >
    Все права защищены © 2020
  </p>
</div>
`;
